src/applications/personalization/profile/components/notification-settings/NotificationItem.jsx

```diff
@@ -86,6 +99,16 @@ const NotificationItem = ({ channelIds, itemName, description, itemId }) => {
     [channelsByItemId],
   );
 
+  // need to do this otherwise we will see Appointment Reminder and Shipment item title only without any checkbox
+  const getMobilePhone = useSelector(state => selectVAPMobilePhone(state));
```

Comment:
nit: I think you can just pass the selector function here without the anonymous function, since `useSelector` will provide `state`:
```
const getMobilePhone = useSelector(selectVAPMobilePhone)
```